source/user-manual/capabilities/system-inventory/compatibility-matrix.rst

```diff
@@ -25,3 +25,27 @@ The Syscollector module supports different options across various operating syst
 +------------------------+-----------+-----------+-----------+----------+-----------+-----------+-----------+
 |    Solaris             |     ✗     |     √     |     √     |     √    |     ✗     |     ✗     |     –     |
 +------------------------+-----------+-----------+-----------+----------+-----------+-----------+-----------+
+
+.. versionadded:: 4.7.0
+
+The Syscollector module now supports scan packages from `NPM (Node Package Manager)  <https://https://www.npmjs.com/>`_, `PyPI (Python Package Index)  <https://https://www.npmjs.com/>`_ and `Snap store <https://snapcraft.io/>`_ , **the support is only for the default paths of those packages' installation**.
+
+The following table shows the new feature that are compatible with various operating systems, where '*√*' means compatible enhancement, '*✗*' not supported enhancement, and '*–*' not applicable.
+
++------------------------+--------------------------------+
+|                        |  **Syscollector enhancement**  |
```

Comment:
I'd simply add that we are talking about an enhancement for the packages scan.
Maybe in the future we could have more tables with improvements for other type of scans.

Reply:
Great. I agree to!